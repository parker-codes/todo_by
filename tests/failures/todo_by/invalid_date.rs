use todo_by::todo_by;

fn main() {
    todo_by!("2024/01/01");
}
