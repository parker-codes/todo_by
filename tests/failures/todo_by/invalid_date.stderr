error: proc macro panicked
 --> tests/failures/todo_by/invalid_date.rs:4:5
  |
4 |     todo_by!("2024/01/01");
  |     ^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Not a valid TODO date: ParseError(Invalid)
